tests/subscription/test_rhsm.py

```diff
@@ -485,20 +111,7 @@ def test_guest_entitlement_status(
         rhsm.attach(host_name=guest_hostname, pool=vdc_pool_physical)
 
 
-@pytest.fixture(scope="class")
-def class_rhsm_sca_disable(rhsm):
-    """Disable sca mode for stage candlepin"""
-    rhsm.sca(sca="disable")
-
-
 @pytest.fixture(scope="class")
 def class_rhsm_sca_enable(rhsm):
```

Comment:
do we still need the sca_enable function?

Reply:
yes, I think now we still the enable action, because the entitlement mode is still not be removed, but we want all the cases run in sca mode, so we have to enable it before testing.
will remove the enable action after it is really removed.

Reply:
okay, make sense